### PR TITLE
Add Event class and Supporting classes

### LINKED ISSUE #280: Code: Create Event objects for Scheduler
Things to do:
- Create the Event class to store the name of the event, time and date.

### PR DESCRIPTION
**Pull request by MatthiasLHK**
- Created Event class
- Created EventName class
- Created EventTime class
- Created EventExceptions package
fixes #280

### CODE REVIEW

File: src/main/java/seedu/address/model/event/EventName.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+public class EventName {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    private final String name;
```
Comment (murtubak) [inline src/main/java/seedu/address/model/event/EventName.java:18]: We should do boundary value testing for this in the future

File: src/main/java/seedu/address/model/event/EventTime.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * Represents the container that stores the start and end time of an Event.
+ */
+public class EventTime {
+    private final LocalDateTime start;
+    private final LocalDateTime end;
+
+    /**
+     * Creates an EventTime object that stores the start and end time.
+     * @param start
+     * @param end
```
Comment (murtubak) [inline src/main/java/seedu/address/model/event/EventTime.java:17]: Could  either elaborate on these parameters or remove them.

**Review by murtubak — APPROVED** [review]
LGTM.

File: src/main/java/seedu/address/model/event/EventName.java (hunk 1 above)
Comment (MatthiasLHK) [reply src/main/java/seedu/address/model/event/EventName.java:18]: Ok noted.

File: src/main/java/seedu/address/model/event/EventTime.java (hunk 2 above)
Comment (MatthiasLHK) [reply src/main/java/seedu/address/model/event/EventTime.java:17]: OK roger, will do boss!

**Pull request merged**